src/services/simulationService.js

```diff
@@ -54,6 +54,8 @@ export class SimulationService {
 
   /** Creates a new simulation */
   static createSimulation(model) {
+    var mod = toSimulationRequestModel(model);
```

Comment:
Doesn't look like this needs to be here. If it does, use `const`.

Reply:
removed, not needed